packages/fxa-auth-server/lib/account-delete.ts

```diff
@@ -300,7 +300,10 @@ export class AccountDeleteManager {
       }
       this.log.error('AccountDeleteManager.deleteFirestoreCustomer', {
         uid,
-        error,
+        error:
+          error instanceof StripeFirestoreMultiError
+            ? error.getSummary()
+            : String(error),
```

Comment:
Does this do something from `error.toString()`?

Reply:
String(error) is same in this case as error.toString() afaik

Reply:
I copied this from the `mozlog` package when I was digging around in it. Seems like `String()` handles undefined/null values where `toString()` would throw a TypeError. ([src](https://stackoverflow.com/questions/3945202/whats-the-difference-between-stringvalue-vs-value-tostring))